### PR TITLE
Connect geth light with auditor node sfdev 7

### PR DESCRIPTION
**Pull request by domagojlatecki**
Set-up Docker Compose for local light Ropsten Ethereum node and fetch stored IPFS file hash from test contract.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,13 +20,19 @@ Kotlin/JavaScript interoperability via GraalVM: https://www.graalvm.org/referenc
 Polyglot programming: https://www.graalvm.org/reference-manual/polyglot-programming/  
 JS compatibility: https://www.graalvm.org/reference-manual/js/JavaScriptCompatibility/  
 
-### IPFS server
-You can either download [IPFS](https://ipfs.io/) desktop application and start it manually or run IPFS via Docker
-Compose. To start IPFS via Docker Compose, position yourself into the `docker` directory and run `docker-compose up -d`.
+### IPFS/Geth nodes
+You can either download [IPFS](https://ipfs.io/) desktop application and [Geth](https://geth.ethereum.org/) and  start
+them manually or them via Docker Compose.
```
Comment (githinho) [inline README.md:25]:
```suggestion
You can either download [IPFS](https://ipfs.io/) desktop application and [Geth](https://geth.ethereum.org/) and start
them manually or run them via Docker Compose.
```

File: README.md [hunk 2]
```diff
@@ -20,13 +20,19 @@ Kotlin/JavaScript interoperability via GraalVM: https://www.graalvm.org/referenc
 Polyglot programming: https://www.graalvm.org/reference-manual/polyglot-programming/  
 JS compatibility: https://www.graalvm.org/reference-manual/js/JavaScriptCompatibility/  
 
-### IPFS server
-You can either download [IPFS](https://ipfs.io/) desktop application and start it manually or run IPFS via Docker
-Compose. To start IPFS via Docker Compose, position yourself into the `docker` directory and run `docker-compose up -d`.
+### IPFS/Geth nodes
```
Comment (githinho) [inline README.md:23]:
```suggestion
 ### IPFS & Geth nodes
```

File: README.md [hunk 3]
```diff
@@ -40,4 +46,12 @@ argument. Current version of the script file should produce the hash value of
 While the Docker IPFS container is running, you can access its web-ui via `http://localhost:5001/webui`.  
 
 If you are running the desktop IPFS application, you can upload the file through its interface. The hash of the file
-will then be displayed there.
+will then be displayed there.  
+
+When the application is started without any program arguments, it will first try to fetch IPFS file hash stored in the
```
Comment (githinho) [inline README.md:51]: Can we change the logic of starting the application, if we don't provide arguments connect to infura and ipfs-gateway and if we provide an argument: `local` or `docker` then connect to local nodes. 
This way user doesn't have to know additional info about starting docker to run the application correctly but if he wants local nodes he needs to set them explicitly.

Comment (domagojlatecki) [reply README.md:51]: Sure, will do

**Review by githinho — APPROVED** [review]

**Pull request merged**